### PR TITLE
Add a menu with links to the sites for previous years

### PR DESCRIPTION
**Pull request by adrianbanks**

### CODE REVIEW

**Comment by alastairs** [comment]
The build log indicates the failure is:

```
[19:21:12][Step 1/1]        "C:\TeamCity\buildAgent\work\77c9f48fc195cd03\DDDEastAnglia.sln" (Build target) (1) ->
[19:21:12][Step 1/1]        "C:\TeamCity\buildAgent\work\77c9f48fc195cd03\DDDEastAnglia\DDDEastAnglia.csproj" (default target) (2) ->
[19:21:12][Step 1/1]        (CoreCompile target) -> 
[19:21:12][Step 1/1]          Controllers\GravatarController.cs(8,30): warning CS0108: 'DDDEastAnglia.Controllers.GravatarController.Url(string, bool, int)' hides inherited member 'System.Web.Mvc.Controller.Url'. Use the new keyword if hiding was intended. [C:\TeamCity\buildAgent\work\77c9f48fc195cd03\DDDEastAnglia\DDDEastAnglia.csproj]
[19:21:12][Step 1/1] 
[19:21:12][Step 1/1] 
[19:21:12][Step 1/1]        "C:\TeamCity\buildAgent\work\77c9f48fc195cd03\DDDEastAnglia.sln" (Build target) (1) ->
[19:21:12][Step 1/1]        "C:\TeamCity\buildAgent\work\77c9f48fc195cd03\DDDEastAnglia.Tests\DDDEastAnglia.Tests.csproj" (default target) (4) ->
[19:21:12][Step 1/1]        (CoreCompile target) -> 
[19:21:12][Step 1/1]          ConferenceLoaderBuilder.cs(61,40): error CS0161: 'DDDEastAnglia.Tests.ConferenceLoaderBuilder.WithTotalNumberOfSessions(int)': not all code paths return a value [C:\TeamCity\buildAgent\work\77c9f48fc195cd03\DDDEastAnglia.Tests\DDDEastAnglia.Tests.csproj]
[19:21:12][Step 1/1] 
[19:21:12][Step 1/1]     1 Warning(s)
[19:21:12][Step 1/1]     1 Error(s)
```

File: DDDEastAnglia/Views/Shared/_Footer.cshtml [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
         <i class="icon-twitter"></i> <a href="http://twitter.com/DDDEastAnglia" target="_blank">Twitter</a>
         &mdash;
-        <span><i class="icon-facebook"></i> <a href="https://www.facebook.com/DDDEastAnglia" target="_blank">Facebook</a></span>
+        <i class="icon-facebook"></i> <a href="https://www.facebook.com/DDDEastAnglia" target="_blank">Facebook</a>
         &mdash;
         <i class="icon-github"></i> <a href="https://github.com/dddeastanglia/DDDEastAnglia/issues" target="_blank">Found a problem?</a>
         &mdash;
-        <i class="icon-globe"></i> <a href="http://dddeastanglia2013.azurewebsites.net">2013 Site</a>
-    </p>
+        <i class="icon-globe"></i>
```
Comment (alastairs) [inline DDDEastAnglia/Views/Shared/_Footer.cshtml]: Should this be inside the `div` that follows it? It seems to me that it's part of the menu item, but perhaps not. Do you have a screenshot to hand or similar?

Comment (adrianbanks) [inline DDDEastAnglia/Views/Shared/_Footer.cshtml]: No, as with it inside it appears on a new line.

**Comment by adrianbanks** [comment]
Re: build failure. This was due to a broken merge on production. I've fixed that separately.

**Pull request merged**